app/views/games/new.html.erb

```diff
@@ -4,7 +4,8 @@
   <div class="col-md-6 col-md-offset-3">
     <h3 class ="text-center">Create A New Game</h3>
     <%= simple_form_for @game, :url => games_path do |f| %>
-      <%= f.input :game_name, :label => 'Game Name', :class => 'form-control' %>
+      <%= f.input :game_name, label: 'Game Name', class: 'form-control' %>
+      <%= f.input :creator_plays_as_black, as: :boolean, label: 'Play as Black', class: 'form-control' %>
```

Comment:
this is the player color selection implementation as a checkbox. could change it to a drop-down later if people like that better.